google/cloud/aiplatform_v1beta1/services/dataset_service/async_client.py

```diff
@@ -996,7 +996,7 @@ async def list_annotations(
         # and friendly error handling.
         rpc = gapic_v1.method_async.wrap_method(
             self._client._transport.list_annotations,
-            default_timeout=None,
+            default_timeout=5.0,
```

Comment:
Just an FYI, now that Bazel is being used the timeout values are pulled from the service config. If the timeouts seem too long/short please edit the service config.

https://github.com/googleapis/googleapis/blob/master/google/cloud/aiplatform/v1beta1/aiplatform_grpc_service_config.json

Reply:
👍